src/inventory_ui.cpp

```diff
@@ -1386,6 +1386,7 @@ void inventory_column::collate()
             if( e->is_item() && e->get_category_ptr() == outer->get_category_ptr() &&
                 e->any_item()->is_favorite == outer->any_item()->is_favorite &&
                 e->any_item()->typeId() == outer->any_item()->typeId() &&
+                !e->any_item()->contents_linked && !outer->any_item()->contents_linked &&
```

Comment:
Why? This seems like a good use case for collation.

<details>
<summary>With the current exclusion:</summary>

![with](https://user-images.githubusercontent.com/68240139/236373557-5ad4b3f1-d020-4bf3-b9c6-bd9cc4d3e6b0.png)

Imagine having 10 of them
</details>


<details>
<summary>Without it:</summary>

![Screenshot from 2023-05-05 06-43-58](https://user-images.githubusercontent.com/68240139/236373580-4d2e479a-31d7-46f8-bd9e-217e55dc79b7.png)

![Screenshot from 2023-05-05 06-44-01](https://user-images.githubusercontent.com/68240139/236373585-c3ee9988-7249-4037-9e35-95fc3e6d8dcb.png)


</details>

Reply:
I intended to keep them separated in two separate collated groups so you could reliably do one or the other, but I didn't actually intend to make all plugged in devices completely separate like that. It's obviously overly strict, now that I'm looking back at it. Replacing the line with `e->any_item()->contents_linked == outer->any_item()->contents_linked &&` makes it work as I originally intended. I need to make it work like that for cable items like extension cords, too.

Also, the battery number showing the entire connected power supply is also unintentional, it should just be showing the device's battery so you can tell how charged it is. I'll take a look at that.